src/user-utilities.js

```diff
@@ -8,7 +8,8 @@ function createUser(firstName, lastName, email, phoneNumber) {
 
     //returns formatted phoneNumber
     function getFormattedPhoneNumber() {
-
+        let p = phoneNumber;
+        return p.slice(0,3)+"-"+p.slice(0,6)+"-"+p.slice(6,11);
```

Comment:
Could use more spacing between operators to make it a bit more clear. p variable might not be very clear as well.

Reply:
@omarmoreta I fixed this could you take another look and see if I need to fix anything else?